docs/modules/user/pages/quickstart.adoc

```diff
@@ -133,6 +132,7 @@ docker run \
     -v nvidia-driver-vol:/usr/nvidia:rw \
     -e HOST_APPS_STATE_FOLDER=/etc/wolf \
     -v /etc/wolf:/etc/wolf:rw \
+    -v /etc/wolf/cfg:/etc/wolf/cfg:rw \
```

Comment:
Instead of adding an additional mount point I think it would be easier to add a line in here https://github.com/games-on-whales/wolf/blob/66e54eb5fc9afa7591d69118e16aeae0e57d405c/docker/startup.sh#L1-L8
Something like 
```bash
mkdir -p $WOLF_CFG_FOLDER
```
The docker run command is big enough already 😅

Reply:
Ah indeed ! Wouldn't it be worth it to add it directly into the CPP code handling default config? Like here: https://github.com/games-on-whales/wolf/blob/66e54eb5fc9afa7591d69118e16aeae0e57d405c/src/moonlight-server/state/configTOML.cpp#L95

Reply:
Yeah you are right, it'd probably be better to do it in Wolf instead. Would you like to add it to your PR? It should be as simple as
```c++
std::filesystem::create_directories(utils::get_env("WOLF_CFG_FOLDER", "/etc/wolf/cfg"));
```
somewhere here https://github.com/games-on-whales/wolf/blob/66e54eb5fc9afa7591d69118e16aeae0e57d405c/src/moonlight-server/wolf.cpp#L427-L430

Otherwise, we can merge the PR and I'll add the check

Reply:
I'm not familiar with cpp but I'll try to !

Reply:
I tried and did not succeed in compiling CPP code sorry - there are lots of dependencies to install just to compile the project, since In run on NixOS I can't just run the commands provided in developer guide (though it's well written) and wont have time to install everything right now. 

I see you have an active Nix branch, having a `flake.nix` with development shell might help people (including myself) to contribute ! If I have time I'll try to contribute this Flake development shell :)

Reply:
Added `mkdir` in container startup script as suggested in the meantime. I guess it should be easy for you to add the auto-dir create in CPP. Shall I create a small issue?

Reply:
Don't worry, I've got this!   
Thanks for taking the time to contribute and for this PR, looks good to me!